### PR TITLE
[INT-1535] Moved code to oauth-function-template directory

### PR DESCRIPTION
**Pull request by agataHappy**
**Ticket** 
https://dronedeploy.atlassian.net/browse/INT-1535

**Work done**
Moved code of oauth-function-template to directory `oauth-template-function`

### CODE REVIEW

**Review by lucasrabiec — CHANGES_REQUESTED** [review]
Please add README in root depth of repo with short information about that what this repo contains.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Functions' Templates
+This is the monorepo for templates that are used in various OAuth functions
```
Comment (lucasrabiec) [inline README.md]: Let's remove `OAuth`. I think this repo can be not only for OAuth functions.

**Review by lucasrabiec — APPROVED** [review]
Almost LGTM, please adjust text in README :+1:

File: README.md (hunk 1 above)
Comment (agataHappy) [reply README.md]: Done

**Pull request merged**